docs/source/getting_started/advanced.rst

```diff
@@ -380,11 +395,11 @@ not have noticed the representation collapse in the first run and continued
 training, using up valuable time and compute resources.
 
 
-Extracting specific Video Frames
+Extracting Specific Video Frames
 --------------------------------
 
 When working with videos, it is preferred not to have to extract all 
-the frames beforehand. With lightly we can not only subsample the video 
+the frames beforehand. With Lightly we can not only subsample the video 
 to find interesting frames for annotation but also extract only these frames.
 
```

Comment:
Can we also remove this section completely?
It does not make that much sense to me:
- Lightly is not designed to be a  video frame loader
- The code looks like is if from an outdated workflow: `read the list of filenames (e.g. from the Lightly Docker output)`

Reply:
I agree, let's remove it.